config.example.wikimedia.yaml

```diff
@@ -39,7 +39,7 @@ wikimedia.org: &wikimedia.org
     - path: projects/wikimedia.org.yaml
       options:
         <<: *default_options
-        pageviews:
+        metrics:
```

Comment:
For the time being, I think we should leave the name as-is, since changing the name requires us to do a RB config change which slows down the deployment unnecessarily.